.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+### Expected behavior:
```

Comment:
To me this template assumes that the issue will be in the form of a bug fix, and wouldn't be appropriate for a feature request.

Reply:
I've updated it with two sections, one for bugs one for features. Thoughts?